### ISSUE TITLE
Error on cookies in console

### ISSUE THREAD

**Issue by justoverclockl** [malago86/flarum-linkpreview#4]
with this new version i get this:

![image](https://user-images.githubusercontent.com/79002016/126171622-30bf3cff-ed13-42ef-89aa-6982ab8a367a.png)

**Comment by malago86** [comment]
Is it also not rewritting the links? I don't see them on the left. Do you get this also on your online forum? It could be something related with the localhost requests... I will take a look at it.

**Comment by justoverclockl** [comment]
Yes also link are not rewrited, thanks to get into this. Maybe is the localhost, but previous works fine

**Comment by malago86** [comment]
I tried a little tweak to the request. Please update to 0.2.1. I am not sure if this will fix it, it does not happen on my browser.

If it still happens, it looks like it might be some issue with the Heroku applet that @Stylix58 created. It might need some extra setup to get it working on cross-domain requests.

However, I am not sure why it happens on your browser and not on mine. I tried both Chrome and Edge.

**Comment by justoverclockl** [comment]
i think i need to try this on a live site, but i can't use this in production right now. same issue

**Comment by stilic** [comment]
We can fix it: just send a PR to https://github.com/Stylix58/meta-grabber.
